examples/platform/esp32/PigweedLogger.cpp

```diff
@@ -0,0 +1,82 @@
+/*
+ *
+ *    Copyright (c) 2021 Project CHIP Authors
+ *
+ *    Licensed under the Apache License, Version 2.0 (the "License");
+ *    you may not use this file except in compliance with the License.
+ *    You may obtain a copy of the License at
+ *
+ *        http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *    Unless required by applicable law or agreed to in writing, software
+ *    distributed under the License is distributed on an "AS IS" BASIS,
+ *    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *    See the License for the specific language governing permissions and
+ *    limitations under the License.
+ */
+
+#include <FreeRTOS.h>
+#include <pw_hdlc/encoder.h>
+#include <pw_stream/sys_io_stream.h>
+#include "semphr.h"
+
+namespace PigweedLogger{
+namespace {
+
+constexpr uint8_t kLogHdlcAddress = 1;   // Send log messages to HDLC address 1 (other than RPC communication)
+constexpr size_t kWriteBufferSize = 128; // Buffer for constructing HDLC frames
+
+SemaphoreHandle_t esp_log_mutex;
+
+static pw::stream::SysIoWriter sWriter;
```

Comment:
nit: Not need for static in unnamed namespace.

Reply:
Done.

Reply:
Could you push the new PR version? This item was marked as done 14 days ago however I still see `static`

Reply:
https://github.com/project-chip/connectedhomeip/pull/5212#discussion_r598903045

Reply:
The request was effectively 'please remove static specifier from this variable declaration. static is not required when using an anonymous namespace since those have static linkage automatically'

Please remove the static specifier from this and the variables and send method below.

Reply:
Done

Reply:
FYI. Similar implementation for other platforms has used a `static` specifier for the above namespace, variables, and send method.